src/base/SolutionArray.cpp

```diff
@@ -35,9 +35,29 @@ const std::map<std::string, std::string> aliasMap = {
     {"Q", "vapor-fraction"},
 };
 
+const std::map<std::string, std::string> reverseAliasMap = {
+    // reserved names used by states
+    {"temperature", "T"},
+    {"pressure", "P"},
+    {"density", "D"},
+    // reserved names used for 1-D objects
+    {"spread-rate", "spread_rate"},
```

Comment:
As an aside: `spread_rate` uses snake_case, while other components don't. Since we're already renaming components, this could also be updated here.